src/extension/src/__main__.py

```diff
@@ -49,6 +49,7 @@ def main(argv):
         runtime_context_handler = RuntimeContextHandler(logger)
         json_file_handler = JsonFileHandler(logger)
         ext_env_handler = ExtEnvHandler(json_file_handler)
+        ext_env_handler.telemetry_supported = telemetry_writer.is_agent_compatible()
```

Comment:
Add a test where telemetry is not supported because env var is not available from agent

Reply:
Comments on the these tests left in TestActionHander